src/widgets/blocks/RegistrationForm.ts

```diff
@@ -89,8 +90,9 @@ export default abstract class RegistrationForm<T extends WidgetConfig> extends W
     };
   }
 
-  protected getCountries(): Array<[string, string]> {
-    const codes = ['AF', 'AL', 'DZ', 'AS', 'AD', 'AO', 'AI', 'AQ', 'AG', 'AR', 'AM', 'AW', 'AU',
+  protected getCountries(config: RegistrationPageConfig): Array<[string, string]> {
+    const codes = config.countryCodes ? config.countryCodes : 
```

Comment:
It's better to rename `codes` to `defaultCodes` and then apply if-then statement to get `codes`. Makes the code easier to read.